docs/source/topics/events.rst

```diff
@@ -70,6 +70,33 @@ and configure ``every_hour`` to be invoked once an hour,
 and ``every_two_hours`` to be invoked once every two hours.
 
 
+.. _cwe-events:
+
```

Comment:
We need to also make sure to update the `api.rst`: https://github.com/aws/chalice/blob/master/docs/source/api.rst for this as well for reference documentation on the new decorator.